scripts/globals/weaponskills.lua

```diff
@@ -638,9 +638,9 @@ function getRangedHitRate(attacker,target,capHitRate,bonus)
 end
 
 function fTP(tp,ftp1,ftp2,ftp3)
-    if (tp < 1000) then
-        tp = 1000
-    end
+    --if (tp < 1000) then
```

Comment:
If these aren't being used anymore, they can just be deleted 👍

Reply:
I think the intent was to floor at the minimum tp you could have in order to ws "just in case" some server decided to let you ws with less. I have no idea if thats even a problem but if it is I think that should be taken care of by whoever is customizing :)

Reply:
And I believe it's not a necessary check either, as this function defaults to returning an ftp of 1 for invalid TP values _anyway_.

So I support the outright removal of the comment parts, @Xlaits ! 👍